edalize/flows/sim.py

```diff
@@ -23,6 +24,8 @@ class Sim(Generic):
     def configure_tools(self, flow):
         if self.flow_options.get("cocotb_module"):
             tool = self.flow_options.get("tool")
+            share_dir = os.popen("cocotb-config --share").read().rstrip() \
```

Comment:
I wouild prefer evaluating `cocotb-config --share` directly like we do with `cocotb-config --libdir`. That makes the script more portable if we want to run it on a different machine than it was created on.

Reply:
fixed by changing the share directory evaluation to a direct call in tools/verilator.py rather than a system call. see commit https://github.com/olofk/edalize/pull/409/commits/aa7d8c31b19fb11f7f73b4ecf312dab157153e7c.